### DISCUSSION TITLE
Error After Updating Library

### DISCUSSION THREAD

**Discussion by syedadil19** [pschatzmann/arduino-audio-tools#1051] [Q&A] [answered]
Hi Phil, I was testing my BLE microphone transmission code with your updated src/Sandbox/BLE library but it returns the following error:
In file included from c:\Users\syeda\Documents\Arduino\libraries\arduino-audio-tools-main\src/Sandbox/BLE/AudioBLE.h:7,
                 from C:\Users\syeda\Downloads\PWM_generator\PWM_generator\PWM_generator.ino:14:
c:\Users\syeda\Documents\Arduino\libraries\arduino-audio-tools-main\src/Sandbox/BLE/AudioBLEClientESP32.h:170:8: error: 'void audio_tools::AudioBLEClient::onScanResult(BLEAdvertisedDevice)' marked 'override', but does not override
   void onScanResult(BLEAdvertisedDevice advertisedDevice) override {
        ^~~~~~~~~~~~

exit status 1

Compilation error: exit status 1

My code was working fine with the previous version of your library before you updated the Sandbox. Would it be fine for me to keep using the older version of your library or would you happen to know why I'm receiving this error?

**Comment by pschatzmann** [comment, marked as answer]
I committed the correction.
Is there any chance that you help me with porting this functioinality to the more generic ArduinoBLE library ?

**Reply by syedadil19** [reply]
Thank you for your help Phil. My project is currently at its peak and I can't give you any promises. I'll let you know as soon as my project ends. Thank you for your understanding.
